ui/components/MesheryMeshInterface/PatternService/CustomTextTooltip.js

```diff
@@ -0,0 +1,21 @@
+import { withStyles } from "@material-ui/styles";
+import { Tooltip } from "@material-ui/core";
+
+export const CustomTextTooltip = ({ backgroundColor, flag, ...props }) => {
+  const CustomTooltip = withStyles(() => ({
+    tooltip : {
+      backgroundColor : backgroundColor,
+      color : "#fff",
+      opacity : "100%",
+      fontSize : "0.75rem",
+      fontFamily : flag ? "Qanelas Soft, sans-serif" : "inherit" ,
+      borderRadius : "0.9375rem",
+      padding : "0.9rem",
+    },
+    popper : {
+      zIndex : "99999 !important"
```

Comment:
No ziCalc()?

What else do we have at 9999? @Abhishek-kumar09 ?

Reply:
We haven't defined the ziCalc function anywhere in meshery ? 
```js
export const ziCalc = (p = 1) => {
  if (p >= 1) {
    // 1 is the least power for zIndex
    let zIndex = "";
    for (let i = 0; i < p; i++) {
      zIndex = zIndex + "9";
    }
    return zIndex;
  }
  `0`;
};

```

Reply:
What special this does apart from having zIndex:"99999" ? // @Abhishek-kumar09

Reply:
nothing. Just creating a  center reference for z-indexes used, so that we dont over/under use the values of z-index.
You don't know how the z-indexes were when this code was not there. people were using 100000000001, etc, etc, or any random numbers. With this we make sure that the least ziCalc value that suffices the UI needs is used and that too don't just use any random numbers.

Reply:
Thanks @Abhishek-kumar09

Reply:
Re-request review when zicalc been included.

Reply:
What @Abhishek-kumar09 said is what we want in principle. This is desired. I just fail to see how the current form of this function accomplishes this.

Does ziCalc(5) return 45?